install/mod_pagespeed_test/unauthorized/inline_css.html

```diff
@@ -1,7 +1,7 @@
 <html>
   <head>
     <title>inline_css example</title>
-    <link rel="stylesheet" href="http://cse.google.com/cse/style/look/default.css">
+    <link rel="stylesheet" href="http://modpagespeed.com/testfiles/google-cse-default.css">
```

Comment:
I think this might defeat the purpose of this test, which is about the behavior of the system with resources on unauthorized domains.  But I guess it's OK because this isn't in an example that would be run with the main page on modpagespeed.com; it would be run from whatever instance of mod_pagespeed was being tested.  Maybe worth a comment though.